ddtrace/contrib/aiokafka/__init__.py

```diff
@@ -0,0 +1,39 @@
+"""
```

Comment:
This docstring is default and should be replaced with one that applies to aiokafka.